boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_iron.dts

```diff
@@ -6,9 +6,25 @@
 
 #include "nrf54h20dk_nrf54h20_cpuapp.dts"
 
-/delete-node/&cpurad_rx_partitions;
+/* The code partitions are re-defined below, so delete the parent node. */
 /delete-node/&cpuapp_rx_partitions;
 
+/* No radio support for IRON variant, don't allocate MRAM. */
+/delete-node/&cpurad_rx_partitions;
+
+/* All of ram0x goes to cpuapp, delete these nodes to re-define below. */
+/delete-node/&cpuapp_ram0x_region;
+/delete-node/&cpurad_ram0x_region;
+
+/* IPC mechanism for IRON variant differs, delete the old node. */
+/delete-node/&cpusec_cpuapp_ipc;
```

Comment:
Doesn't this break PSA?

https://github.com/nrfconnect/sdk-zephyr/pull/2726/files

Reply:
Ah yes, I need to fix this now.